File: RollingDiceGame.ex
b = Enum.random(1..6)
c = Enum.random(1..6)
IO.puts("Welcome To The Rolling Dice Game")
IO.puts("Where To Go?")
IO.puts("1. Start Game")
IO.puts("2. Credits")
a = IO.gets("")
case a do
  "1\n" ->
  IO.puts("You Rolled The First Dice,Number: " <> to_string(b))
  IO.puts("You Rolled The Second Dice,Number: " <> to_string(c))
  IO.puts("The Total Is: " <> to_string(b + c))
  IO.puts("Thanks For Playing The Game!!!")
  "2\n" -> 
  IO.puts("Created By Rabia Alhaffar In 2/November/2019")
  IO.puts("Written In And Powered By Elixir Programming Language!!!")
end

File: RollingDiceGame.rb
b = rand(1..6) 
c = rand(1..6)
puts "Welcome To The Rolling Dice Game!!!"
puts "Where To Go?"
puts "1. Start Game"
puts "2. Credits"
a = gets.chomp
if a.to_i == 1 then
puts "You Rolled The First Dice,Number: #{b}"
puts "You Rolled The Second Dice,Number: #{c}"
puts "The Total Is: #{b + c}"
puts "Thanks For Playing The Game!!!"
elsif a.to_i == 2 then
puts "Created By Rabia Alhaffar In 2/December/2019"
puts "Written In And Powered By Ruby Programming Language!!!"
end